docs/tutorials/Upgrading_to_Application_Commands.md

```diff
@@ -0,0 +1,309 @@
+2. **Verifying request signatures**: To ensure that requests are coming from Discord, your endpoint must verify each request using the included headers, specifically `X-Signature-Ed25519` and `X-Signature-Timestamp`. If the signature fails validating, your app should return a `401` response. More information and example code can be found in the [Security and Authorization documentation](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/security-and-authorization).
+
+> info
+> Many libraries on the [Community Resources page](#DOCS_TOPICS_COMMUNITY_RESOURCES/interactions) simplify verification and interaction request handling by exporting reusable functions and/or handling it automatically.
+
+After your URL is set up to handle signature verification and `PING` requests, you can add your Interaction Endpoint URL by navigating to your app settings from the [developer portal](https://discord.com/developers/applications). On the **General Information** page, you’ll see a field for your **Interactions Endpoint URL**. 
+
+![Interactions endpoint URL in app settings](interactions-url.png)
+
+After you paste your URL and click **Save Changes**, Discord will attempt to verify your endpoint.
+
+If all goes well, your endpoint will successfully save. And if not, you should double check that your app is running with a public endpoint, is verifying request signatures, and is properly responding to `PING` requests.
+
+### Parsing command payloads
+
+Once your app has a verified endpoint, you should start being able to receive command requests from Discord.
+
+As mentioned above, these include information relevant to handling the command like its name, the user who invoked it, and the guild and channel it was invoked from. It also includes additional details that could be relevant, like the [command options](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-structure) or [locale information](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/localization).
+
+Since slash commands (`CHAT_INPUT` commands) are run in the context of a channel, you’ll notice that their payloads don’t contain any information about specific messages. If your app needs the message content, you can use [message commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/message-commands) which *do* include the message content.
+
+> info
+> In the getting started guide’s repository, there’s a code sample showing [how to create and handle commands using JavaScript](https://github.com/discord/discord-example-app/blob/main/examples/command.js).
+
+Below is an example payload your app would receive when a user invoked a global command with an option:
+
+```
```

Comment:
examples throughout the documentation contain actual values resembling ones you see in practice, maybe they should be here too?

Reply:
I thought for this payload specifically, it might be helpful to notate the payload instead so it's clear which nested `id` fields are important to pay attention to, and how different fields map to features/concepts